COLLADASaxFrameworkLoader/src/COLLADASaxFWLMeshLoader.cpp

```diff
@@ -913,11 +920,13 @@ namespace COLLADASaxFWL
                 mColorList.push_back ( color );
             }
         }
+		return false;
     }
 
     //------------------------------
-    void MeshLoader::initializeTexCoordsOffset ()
+    bool MeshLoader::initializeTexCoordsOffset ()
```

Comment:
same as above, the function calls the application error handler, so either avoid that call or report back the error handler's decision for continue parsing or abort.